packages/wpcom-checkout/src/transformations.ts

```diff
@@ -280,12 +287,48 @@ function getYearlyVariantFromProduct( product: ResponseCartProduct ) {
 	return product.product_variants.find( ( variant ) => 12 === variant.bill_period_in_months );
 }
 
+export function hasProductFreeMonthTrial( product: ResponseCartProduct ): boolean {
+	return (
+		product.bill_period === '365' &&
+		product.introductory_offer_terms?.interval_unit === 'month' &&
+		product.introductory_offer_terms?.interval_count === 1 &&
+		product.item_subtotal_integer === 0
+	);
+}
+
 export function getSubtotalWithoutDiscounts( responseCart: ResponseCart ): number {
 	return responseCart.products.reduce( ( total, product ) => {
+		const isJetpack = isJetpackProduct( product ) || isJetpackPlan( product );
+
+		if ( isJetpack && hasProductFreeMonthTrial( product ) ) {
+			return (
+				total + product.item_original_subtotal_integer + product.item_original_monthly_cost_integer
+			);
+		}
+		if ( isJetpack && isBiennially( product ) ) {
+			const yearlyVariant = getYearlyVariantFromProduct( product );
+
+			if ( yearlyVariant ) {
+				return total + yearlyVariant.price_before_discounts_integer * 2;
+			}
+		}
+
 		return total + product.item_original_subtotal_integer;
 	}, 0 );
 }
 
+export function getSubtotalWithDiscounts( responseCart: ResponseCart ): number {
+	return responseCart.products.reduce( ( total, product ) => {
+		const isJetpack = isJetpackProduct( product ) || isJetpackPlan( product );
+
+		if ( isJetpack && hasProductFreeMonthTrial( product ) ) {
+			return total + product.item_original_subtotal_integer;
```

Comment:
Can you explain why this change exists? If the subtotal is wrong, it seems like something we should fix on the backend, but I'm sure there's something I'm just misunderstanding.

Reply:
In the backend, it's correct that the first introductory term will be $0. However, from the user perspective we want to achieve the following:
![image](https://github.com/Automattic/wp-calypso/assets/8419292/af38161e-1293-4d98-a268-6a456403b505)

Reply:
I wonder if it would be worth it to add a new value to the backend to handle situations such as this 🤔

Reply:
Oh, I see, so the idea is to have this function return "the subtotal without discounts including multi-year discounts" even though multi-year discounts are not tracked the way we track normal discounts. I didn't mean this was a bad approach, but maybe we should add a comment block to this method or use slightly more explicit naming to explain the situation. Although it looks like maybe https://github.com/Automattic/wp-calypso/pull/86914 handled this in a different way.